client/src/components/ListOfItems/style.css

```diff
@@ -0,0 +1,18 @@
+.card {
+    display: grid;
+    grid-template-rows: auto 1fr auto;
+    grid-gap: 0.25rem;
+    border-radius: 0.25rem;
+    box-shadow: 0 2px 6px hsla(220, 10%, 30%, 0.2),
+        0 8px 16px hsla(220, 10%, 30%, 0.2);
+    padding: 1rem;
+    background-color: #fff;
+}
+
+.grid {
```

Comment:
[BEM](http://getbem.com/introduction/) class naming convention is a very good thing to get used to.